pkg/controller/datadogagent/clusterchecksrunner.go

```diff
@@ -374,6 +370,13 @@ func getEnvVarsForClusterChecksRunner(dda *datadoghqv1alpha1.DatadogAgent) []cor
 		})
 	}
 
+	if spec.Agent.Config.DDUrl != nil {
+		envVars = append(envVars, corev1.EnvVar{
+			Name:  datadoghqv1alpha1.DDddURL,
+			Value: *spec.Agent.Config.DDUrl,
```

Comment:
nit
```suggestion
			Value: datadoghqv1alpha1.BoolValue(spec.Agent.Config.DDUrl),
```
using `datadoghqv1alpha1.BoolValue` would be safer here, we can also use it to get `spec.ClusterChecksRunner.Config.LogLevel` in the block above